server/data/tokenVerification.ts

```diff
@@ -18,7 +18,16 @@ function getApiClientToken(token: string) {
 
 type TokenVerifier = (request: Request) => Promise<boolean | void>
 
+type RequestWithUser = Request & { user: Express.User }
+
+const assertHasUser: (req: Request) => asserts req is RequestWithUser = req => {
+  if (!('user' in req)) {
+    throw new Error('Request object without user found unexpectedly')
+  }
+}
+
 const tokenVerifier: TokenVerifier = async request => {
+  assertHasUser(request)
```

Comment:
This should be importing `assertHasUser` from `typeUtils`, and not redefine the `RequestWithUser` type

Reply:
oops

Reply:
Probably my mistake!